msteams-platform/concepts/build-and-test/share-to-teams-overview.md

```diff
@@ -10,11 +10,14 @@ ms.date: 02/09/2023
 
 Share to Teams is a button enabled on any web app, personal app, or tab that allows users to share content to other users in Teams.
 
+> [!NOTE]
+> Share to Teams isn't available for [Government Community Cloud (GCC) High and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+
 You can also decide the placement of the Share to Teams button on the app. Users can pin the crucial content and refer to it while collaborating with Teams.
 
 :::image type="content" source="../../assets/images/share-to-teams/overview.PNG" alt-text="Screenshot showing the Share button on the app.":::
 
-You can enable this feature on the following:
+You can enable this feature on the following scopes:
```

Comment:
```suggestion
You can enable Share to Teams button in the following:
```

Reply:
Updating as:

You can enable Share to Teams button in the following scopes: